bbq/vm/test/vm_test.go

```diff
@@ -3232,3 +3232,80 @@ func TestIfLet(t *testing.T) {
 		assert.Equal(t, vm.NewIntValue(2), result)
 	})
 }
+
+func TestCompileSwitch(t *testing.T) {
+
+	t.Parallel()
+
+	t.Run("1", func(t *testing.T) {
+
```

Comment:
```suggestion
		t.Parallel()
```

Reply:
Parallelized the other subtests in [54a9df5](https://github.com/onflow/cadence/pull/3757/commits/54a9df59f333ca7041dd948fe0afdef97a096c4e)